content/content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -4215,7 +4215,7 @@ public String buildListContext (	VelocityPortlet portlet,
 			// notshow the public option or notification when in dropbox mode
 			context.put("dropboxMode", Boolean.TRUE);
 			// allow filtering of dropboxes by group (SAK-14625)
-			String collectionId = (String) state.getAttribute (STATE_COLLECTION_ID);
```

Comment:
It's just I don't like to remove code of others, I prefer to comment just in case the variable was put there as a reminder or something. I'll delete it.